test/integration/targets/storage_gateway/defaults/main.yaml

```diff
@@ -0,0 +1,39 @@
+---
+# defaults file for ec2_instance
+ec2_instance_name: '{{resource_prefix}}-node'
+ec2_instance_owner: 'integration-run-{{resource_prefix}}'
+ec2_file_ami_image:
+  # amazon/aws-storage-gateway-file-2018-03-21
```

Comment:
If there's a way to filter for these you could use ec2_ami_facts so we always get the latest ones.